banyand/observability/metrics_system.go

```diff
@@ -152,3 +161,22 @@ func getNetStat(ctx context.Context) ([]net.IOCountersStat, error) {
 	}
 	return availableStats, nil
 }
+
+func collectUpTime() {
+	upTimeGauge.Set(time.Since(startTime).Seconds())
+}
+
+func collectDisk() {
+	for path := range getPath() {
+		usage, err := disk.Usage(path)
+		if err != nil {
+			// skip logging the case where the data has not been created
+			if err.Error() == "no such file or directory" {
+				log.Error().Err(err).Msgf("failed to get usage for path: %s", path)
+			}
+			return
+		}
+		diskStateGauge.Set(usage.UsedPercent/100, path, "used_percent")
+		diskStateGauge.Set(float64(usage.Used)/float64(usage.Total), path, "used")
```

Comment:
Wouldn't "used" be referring to the `usage.Used`?

Reply:
Updated.  Setting "Used" and "Total" separately. Updated for memory as well.